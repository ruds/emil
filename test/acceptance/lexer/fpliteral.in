123.456
-123.456
123.
-123.
123.e78
-123.E78
123.E-78
123.456e78
-123e78
123E-78
-123f
123F
Inf
-Inf
NaN
123_456.000_789
.456
(*xx*)
-.456
(*xx*)
0xcafe.01
(*xx*)
0o777.20
(*xx*)
0B101.010
(*xx*)
