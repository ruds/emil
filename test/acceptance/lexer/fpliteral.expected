0001 FPLITERAL 1.2345600000000e+02 123.456
0002 FPLITERAL -1.2345600000000e+02 -123.456
0003 FPLITERAL 1.2300000000000e+02 123.
0004 FPLITERAL -1.2300000000000e+02 -123.
0005 FPLITERAL 1.2300000000000e+80 123.e78
0006 FPLITERAL -1.2300000000000e+80 -123.E78
0007 FPLITERAL 1.2300000000000e-76 123.E-78
0008 FPLITERAL 1.2345600000000e+80 123.456e78
0009 FPLITERAL -1.2300000000000e+80 -123e78
0010 FPLITERAL 1.2300000000000e-76 123E-78
0011 FPLITERAL -1.2300000000000e+02 -123f
0012 FPLITERAL 1.2300000000000e+02 123F
0013 FPLITERAL inf Inf
0014 FPLITERAL -inf -Inf
0015 FPLITERAL nan NaN
0016 FPLITERAL 1.2345600078900e+05 123_456.000_789
0017 ERROR
0019 ERROR
0021 ERROR
0023 ERROR
0025 ERROR
0027 END
